omnibus/files/private-chef-cookbooks/private-chef/templates/default/postgresql.conf.erb

```diff
@@ -77,7 +77,9 @@ max_connections = <%= node['private_chef']['postgresql']['max_connections'] %>
 # - Security and Authentication -
 
 #authentication_timeout = 1min		# 1s-600s
-#ssl = off				# (change requires restart)
+ssl = <%= (node['private_chef']['postgresql']['sslmode'] == 'disable' ? 'off' : 'on') + ' # (change requires restart)' %>
+ssl_cert_file = <%= "'#{node['private_chef']['postgresql']['ssl_cert_file']}' # (change requires restart)" %> %>
+ssl_key_file  = <%= "'#{node['private_chef']['postgresql']['ssl_key_file']}'  # (change requires restart)" %> %>
```

Comment:
might be useful to add conditionals around these to be set only if the sslmode is set to require.

Reply:
#(change requires restart) seems to be a comment on the line. Not needed to be included in the value of the keys for ssl, ssl_cert_file or ssl_key_file

Reply:
Originally I had conditionals, but removed them because it has no effect anyway if ssl=off, and the lines simply point to cert files which will already exist (after recipe is created to place them).

I can add the conditionals back if you like.  Let me know your thoughts.

Reply:
I think the pattern is to put #(change requires restart) as a comment, do we need it to be a part of that expression?

Reply:
until the recipe to add the default certs exists, I would prefer adding conditionals.
a) to check if sslmode is require and
b) to confirm that those files exist.

Reply:
The comment only needs to be a part of the expression if we want the comment rendered to the output file.  Anyway, I removed it.

Conditionals added.  Will add code to check if files exist.